src/app/containers/ScheduleSummary.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import * as moment from 'moment';
+import { connect } from 'react-redux';
+import { RootState } from '../store';
+import {
+    getSheriffLeaves,
+    getSheriffShifts
+} from '../modules/shifts/selectors';
+import {
+    getLeaves,
+    getShifts
+} from '../modules/shifts/actions';
+import {
+    default as ScheduleSummary,
+    StatusEnum
+} from '../components/ScheduleSummary/ScheduleSummary';
+import {
+    Leave,
+    Shift,
+    IdType,
+    DateType
+} from '../api';
+
+interface ConnectedScheduleSummaryProps {
+    sheriffId: IdType;
+    start: DateType;
+    end: DateType; 
+}
+
+interface ConnectedScheduleSummaryDispatchProps {
+    fetchShifts: () => void;
+    fetchLeaves: () => void;    
+}
+
+interface ConnectedScheduleSummaryStateProps {
+    leaves: Leave[];
+    shifts: Shift[];
+}
+
+class ConnectedScheduleSummary extends React.Component<ConnectedScheduleSummaryProps
+    & ConnectedScheduleSummaryStateProps
+    & ConnectedScheduleSummaryDispatchProps> {
+
+    componentWillMount() {
+        const { fetchLeaves, fetchShifts } = this.props;
```

Comment:
I think we can leave out these fetch requests.  Considering that they will be issuing web requests, how many of these ScheduleSummary Components will be on the page?  In this case, the LongTermSchedule should have already made the web request for getting shifts and leaves so I think we can avoid having to do it here.